From: Lucas Kanashiro <kanashiro@ubuntu.com>
Date: Fri, 5 Nov 2021 11:36:10 -0300
Subject: Add ruby3.0 support

Origin: backport, https://github.com/vcr/vcr/commit/b38369dcace0
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=996521
---
 lib/vcr/structs.rb | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/lib/vcr/structs.rb b/lib/vcr/structs.rb
index b4a3fc5..ae8e30a 100644
--- a/lib/vcr/structs.rb
+++ b/lib/vcr/structs.rb
@@ -454,9 +454,10 @@ module VCR
 
       case type
       when 'gzip'
-        args = [StringIO.new(body)]
-        args << { :encoding => 'ASCII-8BIT' } if ''.respond_to?(:encoding)
-        yield Zlib::GzipReader.new(*args).read
+        gzip_reader_options = {}
+        gzip_reader_options[:encoding] = 'ASCII-8BIT' if ''.respond_to?(:encoding)
+        yield Zlib::GzipReader.new(StringIO.new(body),
+                                   **gzip_reader_options).read
       when 'deflate'
         yield Zlib::Inflate.inflate(body)
       when 'identity', NilClass
